package.json

```diff
@@ -31,38 +33,32 @@
     "@msgpack/msgpack": "^2.8.0",
     "@ngneat/until-destroy": "^9.2.3",
     "@ngrx/store": "^15.4.0",
-    "@stablelib/hkdf": "^1.0.1",
```

Comment:
@jongbonga Im confused with what happened with the package.json files during the merge which is probably resulting in the conflicts?

Reply:
did you pull the latest changes from dev? where did you branch out from?

Reply:
Most issues are on the package json. I will try and detangle it today